core/Pos/Core/Genesis/Generate.hs

```diff
@@ -163,7 +164,7 @@ generateSecretsAndAddress mbSk hasHDPayload= do
             if not hasHDPayload then makePubKeyAddressBoot (toPublic sk)
             else
                 fst $ fromMaybe (error "generateKeyfile: pass mismatch") $
-                deriveLvl2KeyPair (IsBootstrapEraAddr True) emptyPassphrase hdwSk
+                deriveLvl2KeyPair (IsBootstrapEraAddr True) (ShouldCheckPassphrase False) emptyPassphrase hdwSk
```

Comment:
Can you explain why it's safe to skip checking it here?

Reply:
because we created `hdw` secret in `generateSecrets` with an empty passphrase and I don't see a lot of sense to check it, especially considering generation genesis by genesis spec is a long operation